ci/apply-services

```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
@dagoof you happy to keep the .sample.json files in or is there a better way?

Reply:
i want to leave them in. we need a better solution eventually but they're pretty crucial right now so we at least know what should exist